### ISSUE TITLE
Increasing UI Elements Scaling causes ui to become more squished

### ISSUE THREAD

**Issue by AaronRecord** [aseprite/aseprite#2571]
I'm using a high dpi display (System scaling is 250% and resolution is 3200x1800).
`Screen Scaling` set to 400% and `UI Elements Scaling` set to 100%:
![Screenshot 2020-11-27 205726](https://user-images.githubusercontent.com/50304111/100493379-8ca8f000-30f3-11eb-8c68-07ba214e89a6.png)
![Screenshot 2020-11-27 205708](https://user-images.githubusercontent.com/50304111/100493380-90d50d80-30f3-11eb-8852-d620a3f7206c.png)
`Screen Scaling` set to 100% and `UI Elements Scaling` set to 400%:
![Screenshot 2020-11-27 205945](https://user-images.githubusercontent.com/50304111/100493385-9b8fa280-30f3-11eb-87b5-8e64339ccefa.png)
![Screenshot 2020-11-27 205643](https://user-images.githubusercontent.com/50304111/100493387-9fbbc000-30f3-11eb-900a-184c88b19c2b.png)

### Aseprite and System version

* Aseprite version: 1.2.25, Steam
* System: Windows 10

**Comment by iamOgunyinka** [comment]
There are a few things worth mentioning here, so I would list them, in no particular order. The overall idea is that when you make and apply changes to the UI/Screen scaling from the preference/option window, all elements on the screen from the preference window itself to the menu (bar) down to the minutest element reflects the change (including resizing the window to make room or shrink to fit) without having to close and reopen windows.

Right now, this almost worked for everything except some menu items that become inconsistent (e.g. as shown below)

![image](https://user-images.githubusercontent.com/4146085/149783951-79f0b6f8-5502-4ba7-b69e-ad72644834aa.png)

To solve menu bar(and items) problem, we could simply [reload the menu bar](https://github.com/iamOgunyinka/aseprite/commit/dbc5c2260cdcde2a46ae8405a8c0d278007fb12f#diff-58ea2c4ebff4782780abeb17f1c8da234102027094dcf3b9aa73c55cd8030517R964) and the main window seem to have no additional problem after this has been done. Then focus shifts to the option window exclusively now.

When UI scaling is adjusted, we [reload the theme, apply/init the new theme](https://github.com/iamOgunyinka/aseprite/blob/9e23d31d84a5bfdccc32aea065ffb60b02b57856/src/app/commands/cmd_options.cpp#L767) to all the widgets and reset the [screen](https://github.com/iamOgunyinka/aseprite/blob/9e23d31d84a5bfdccc32aea065ffb60b02b57856/src/app/commands/cmd_options.cpp#L796). `Invalidate()`-ing the option window should ideally force a redraw of this opened window but unfortunately in this case, I tried it and it seem to be ignored. The texts of the list box that shows the [sections](https://github.com/iamOgunyinka/aseprite/blob/9e23d31d84a5bfdccc32aea065ffb60b02b57856/data/widgets/options.xml#L9) are properly scaled and the list box size is adjusted to house the children (list box items). However, the [parent view](https://github.com/iamOgunyinka/aseprite/blob/9e23d31d84a5bfdccc32aea065ffb60b02b57856/data/widgets/options.xml#L8) (or view port) of the listbox seem to be incorrectly calculating the size it needs to house the list box and thus result in _cramping_ them in the littlest space(as shown in the image below). If this parent view port is removed, the items are correctly sized and adjusted.

Another noticeable problem is that all the texts in the option window's combo boxes (of all sections, not just the general section) are still in the previous UI scaling, not the new one. I traced this problem and followed all paths to ensure that really the theme is applied to all the items in the comboboxes and are available when a [combo box is opened](https://github.com/aseprite/aseprite/blob/main/src/ui/combobox.cpp#L614-L645) but I could not get to the root of this.

![image](https://user-images.githubusercontent.com/4146085/149791968-15242760-be16-47f9-9601-0d8a4f5e6e42.png)

tl;dr the two things I could not get to the bottom of: 
1. scaling of list items in option window's combo boxes and
2. getting the section listbox size right.
